decidim-core/lib/decidim/content_blocks/has_related_components.rb

```diff
@@ -34,11 +34,15 @@ def components_for(content_block)
         end
 
```

Comment:
Let's reduce the visibility here, as both of these methods are only used by the `components_for` method as long as I can see: 
```suggestion

        private

```